packages/opentelemetry-node/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+- feat: add env var `ELASTIC_OTEL_HOST_METRICS_DISABLED` to controll wether
+  EDOT has to send metrics from the host (process, system, ...) or not. This
+  means users can disable host metrics without affecting metrics from instrumentations.
+  (https://github.com/elastic/elastic-otel-node/issues/736)
+
+  The existing `ELASTIC_OTEL_METRICS_DISABLED` becomes deprecated since the
+  disabling of metrics can be done via `OTEL_METRICS_EXPORTER` configuration
+  option.
```

Comment:
```suggestion
- The `ELASTIC_OTEL_METRICS_DISABLED` environment variable is now deprecated.
  Use `OTEL_METRICS_EXPORTER=none` (instead of `ELASTIC_OTEL_METRICS_DISABLED=true`)
  to disable any metrics being exported by EDOT Node.js.
```